composer.json

```diff
@@ -11,9 +11,9 @@
         }
     ],
     "require": {
-        "php": ">=5.6.4",
-        "illuminate/queue": "5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*|6.*",
-        "illuminate/support": "5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*|6.*"
+        "php": ">=7.0",
+        "illuminate/queue": "8.*",
```

Comment:
Laravel versions 6+ use a different version selector than 5.x and below since it's now correctly semantic versioning. Could you change this to `^8.0` (for this and the below line)?